azext_edge/edge/providers/rpsaas/adr2/command.md

```diff
@@ -0,0 +1,476 @@
+	--queue-size
+	--topic-path
+	--topic-retention (`keep`, `never`) [may turn this into bool flag]
+
+  export
+
+	--asset
+	--resource-group
+	[optional]
+	--format (`json`, `yaml`, `portal-csv`)
+	--output-dir
+	--replace
+
+  import
+
+	--asset
+	--resource-group
+	--input-file
+	[optional]
+	--replace
+
+  list
+
+	--asset
+	--resource-group
+
+  remove
+
+	--asset
+	--resource-group
+	--event [or --name]
+
+  update
+
+	--asset
+	--resource-group
+	--event [or --name]
+	[optional]
+	--observability-mode (`none`, `log`)
+	--publishing-interval
+	--sampling-interval
+	--queue-size
+	--topic-path
+	--topic-retention (`keep`, `never`) [may turn this into bool flag]
+
+## Asset Endpoint Profiles
+id: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/assetEndpointProfiles/my-assetendpointprofile`
+
+main changes: remove the cert subgroup, modify some create/update params.
+
+Discovered asset endpoint profiles are different but have the same props:
+id: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/discoveredAssetEndpointProfiles/my-assetendpointprofile`
+
+### asset endpoint
+
+  create
+
+	--endpoint-profile [or --name]
+	--instance
+	--resource-group
+	--target-address
+	[optional]
+	--location
+	--instance-resource-group
+	--instance-subscription
+	--additional-configuration
+	--profile-type
+	--certificate-ref [use this, user/password-ref to determine auth-mode]
+	--password-ref
+	--user-ref
+	--tags
+
+  delete
+
+	--endpoint-profile [or --name]
+	--resource-group
+
+  list/query
+
+	[optional]
+	--discovered [bool flag]
+	--resource-group
+	--instance
+	--instance-resource-group [required if instance is used]
+	--target-address
+	--auth-mode
+	--profile-type
+	--resource-query
+
+  show
+
+	--endpoint-profile [or --name]
+	--resource-group
+
+  update
+
+	--endpoint-profile [or --name]
+	--resource-group
+	[optional]
+	--target-address
+	--additional-configuration
+	--profile-type
+	--certificate-ref [use this, user/password-ref to determine auth-mode]
+	--password-ref
+	--user-ref
+	--tags
+
+## Schemas
+
+Need to create schema registry first. Schema is a subresource. Schema version is a subresource of schema.
+
+In the end, the id (for version) looks like this: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/schemaRegistries/my-schema-registry/schemas/my-schema/schemaVersions/1`
+
+custom location is not needed...
+
+### schema
+  create
+
+	--schema [or --name]
+	--schema-registry
+	--resource-group
+	--format
+	--schema-type
+	[optional]
+	--description
+	--display-name
+	--tags
+
+  delete
+
+	--schema [or --name]
+	--resource-group
+
+  show
+
+	--schema [or --name]
+	--resource-group
+
+  list
+
+	--resource-group
+	--schema-registry
+	[optional]
+	--format
+	--schema-type
+	--display-name
+
+### schema registry
+  create
```

Comment:
you can remove create / delete / show / list

Reply:
kk